ods_ci/run_robot_test.sh

```diff
@@ -20,6 +20,8 @@ EMAIL_SERVER_USER="None"
 EMAIL_SERVER_PW="None"
 EMAIL_SERVER_SSL=false
 EMAIL_SERVER_UNSECURE=false
+OPEN_REPORT_IN_BROWSER=false
+REPORT_BROWSER="firefox" # Default browser to open reports in
```

Comment:
Shouldn't this be exposed as well like `--open-report`?

Reply:
hm, well, I can add another flag for it, but as described in comment for `--open-report` you can use following now:

`--open-report true` - Firefox will be used to open the reports
`--open-report false` - nothing is opened after the tests execution
`--open-report google-chrome` - Google Chrome will be used to open the reports
`--open-report non-existent` - this will fail to open anything :slightly_smiling_face:

So at the moment, you don't need to use another flag to define your desired browser/app to open the directory with test reports.

Reply:
Ah, sorry, totally missed that! It's good as it is then.